src/watcher.js

```diff
@@ -7,9 +7,38 @@ import {FileFilter} from './cmd/build';
 
 const log = createLogger(__filename);
 
+// Flow Types
+
+export type ShouldWatchFn = (filePath: string) => boolean;
+
+export type OnChangeFn = () => any;
```

Comment:
unfortunately if we change this into `() => void`, then we need to remove the `return` from the `onChange` function defined in "cmd/build.js" and then "test.build.js" fails because it is currently using the return value of the `onChange` function.